cfme/configure/configuration/__init__.py

```diff
@@ -1445,7 +1445,7 @@ class DatabaseBackupSchedule(Schedule):
         ("time_zone", {
             version.LOWEST: Select("select#time_zone"),
             '5.5': AngularSelect('time_zone')}),
-        ("start_date", Calendar("miq_angular_date_1")),
+        ("start_date", Calendar("start_date")),
```

Comment:
We want to use version pick here - `miq_angular_date_1` for 5.4 and `start_date` for 5.5+.